plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -310,6 +311,10 @@ protected String getTableSchemaName(ResultSet resultSet)
 
             case Types.DATE:
                 return Optional.of(dateColumnMapping());
+
+            case Types.TIMESTAMP:
+                // TODO support higher precisions (https://github.com/trinodb/trino/issues/6910)
+                return Optional.of(timestampColumnMappingUsingSqlTimestampWithRounding(TIMESTAMP_MILLIS));
```

Comment:
OK to remove these line for now, as the intention is to fix the default mapping.

```suggestion
                break;
```